cli/types/index.d.ts

```diff
@@ -64,6 +64,7 @@ declare namespace Cypress {
     path: string
     isHeaded: boolean
     isHeadless: boolean
+    family: string

```

Comment:
don't we already define this interface inside of `packages/launcher`? or somewhere else?

Reply:
probably because there is no clean way to share code between packages and this - a lot of types are duplicated because of that. it would be nice to revisit how we do types to untangle this mess.